docs/dev/project_management.rst

```diff
@@ -8,16 +8,72 @@ management tools as well as `Zenhub <https://www.zenhub.com>`__ which provides
 additional features for sprint planning, task estimation, and progress reports.
 
 -------------------------------------------------------------------------------
-Issues and Project Tracking
+Agile Scrum Kanban Iteration SDLC Process
 -------------------------------------------------------------------------------
-We use `Github issues <https://github.com/catalyst-cooperative/pudl/issues>`__ to
-track bugs, enhancements, support requests, and just about any other work that goes
-into the project. Try to make sure that issues have informative tags so we can find
-them easily.
-
-We use Zenhub Sprints, Epics, and Releases to track our progress. These won't be
-visible unless you have the `ZenHub browser extension
-<https://www.zenhub.com/extension>`__ installed.
+
+We merge code to ``dev`` continuously, but plan work in two-week sprints.
+
+We track work as items (GitHub issues and PRs) in a Catalyst-wide Github Project.
+
+During each sprint, people on each team pull from a list of prioritized & scoped
+items.
+
+Each team has its own independent sprint process.
+
+We have 2 or 3 meetings per sprint:
+
+- sprint planning: overall goal is "get a good idea of what work is coming up
+  in the sprint"
+
+  - mark items with a new sprint
+
+    - enough that that we feel like there's enough work to do
+
+    - few enough that we don't feel overwhelmed
+
+- (Inframundo only - other teams do this asynchronously): mid-sprint check-in:
+  overall goal is "given this sprint's status, prioritize items for the next
+  sprint"
+
+  - make sure item status is up-to-date (this can be done asynchronously and
+    just confirmed in-meeting)
+
+  - identify items that need clarification / re-scoping
+
+  - pull items from the icebox into the backlog
+
+- sprint review: overall goal is "prepare prioritized items for the next sprint"
+
+  - make sure item status is up-to-date (this can be done asynchronously and
+    just confirmed in-meeting)
+
+  - for item in backlog: if item is not ready for development, assign 1 or 2
+    people to flesh it out
+
+An item is considered "ready for development" if has:
+
+- an assigned team (via GitHub label), so the issue will be picked up in
+  *someone*'s sprint process
+
+- a concrete set of criteria that define the scope in the issue description, so
+  we can stop when we want to and also discuss cutting scope productively
+
+- a priority field in the project (Urgent, High, Medium, Low), so we know which
+  things to work on first
+
+- a size field in the project (X-Large, Large, Medium, Small, Tiny), so we know
+  how much time we think an item will take & how likely it is to grow beyond
+  our expectations.
+
```

Comment:
Both urgency and size can feel somewhat arbitrary to apply - it'd be helpful to think of examples or rough guidelines here. E.g., "Large is a 3-week project" or "Urgent is a catastrophic failure type bug vs High is blocking other PRs"